api/schemas/audit_models.py

```
@@ -0,0 +1,427 @@
+"""Modelos Pydantic para auditoria completa de PDFs"""
+from pydantic import BaseModel, Field, validator
+from typing import List, Dict, Optional, Any
```

Ruff (0.14.4)

`typing.List` is deprecated, use `list` instead

(UP035)